pkg-install

```diff
@@ -9,10 +9,12 @@ import subprocess
 parser = ArgumentParser(description="Install AUR packages into a Docker container")
 parser.add_argument("package_name", type=str, help="name of the AUR package")
 parser.add_argument("-x", type=str, metavar='EXECUTABLE', help="name of executable")
+parser.add_argument("-e", type=str, help="edit generic dockerfile")
```

Comment:
The label  "edit generic dockerfile" is kind of misleading since this pull request doesn't allow the user to edit the entire Dockerfile but only to append a RUN command inside of it.

Reply:
Good point, fixed!